libnmstate/nm/bond.py

```diff
@@ -79,14 +80,23 @@ def _get_options(nm_device):
         bond_option_names_in_profile.add("arp_interval")
         bond_option_names_in_profile.add("miimon")
 
+    # Mode is required
+    sysfs_folder = SYSFS_BOND_OPTION_FOLDER_FMT.format(ifname=ifname)
+    mode = _read_sysfs_file(f"{sysfs_folder}/mode")
+    if not mode:
+        return {}
```

Comment:
How is it possible for `mode` not to exist? A bond must have a mode AFAIK.

Reply:
Removed.